src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/international/standing/orders/junit/v3_1_8/GetInternationalStandingOrderInternationalStandingOrderIdPaymentDetailsTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.forgerock.uk.openbanking.tests.functional.payment.international.standing.orders.junit.v3_1_8
+
+import com.forgerock.securebanking.framework.extensions.junit.CreateTppCallback
+import com.forgerock.securebanking.framework.extensions.junit.EnabledIfVersion
+import com.forgerock.securebanking.openbanking.uk.common.api.meta.obie.OBVersion
+import com.forgerock.uk.openbanking.tests.functional.payment.international.standing.orders.api.v3_1_8.GetInternationalStandingOrderInternationalStandingOrderIdPaymentDetails
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class GetInternationalStandingOrderInternationalStandingOrderIdPaymentDetailsTest(val tppResource: CreateTppCallback.TppResource) {
+
```

Comment:
a bit long name.

Reply:
Renamed this class and others as well that were too long.